posts/2023-06-13-23.0.0.6-beta.adoc

```diff
@@ -0,0 +1,174 @@
+blog_description: Open Liberty 23.0.0.6-beta includes support for Private Key JWT authentication and Proof Key for Code Exchange (PKCE) for OpenID Connect clients. This release also includes the latest updates to our ongoing InstantOn functionality, which provides incredibly fast startup times, as well as the latest Jakarta Data preview. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= OIDC Clients gain PKCE and Private Key JWT authentication support in Open Liberty 23.0.0.6-beta 
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 23.0.0.6-beta includes support for Private Key JWT authentication and Proof Key for Code Exchange (PKCE) for OpenID Connect clients.
+
+This release also includes the latest updates to our ongoing link:{url-prefix}/blog/2023/02/10/instant-on-beta-update.html[InstantOn functionality], which provides incredibly fast startup times, as well as the latest link:{url-prefix}/blog/2023/04/18/23.0.0.4-beta.html[Jakarta Data preview]. 
+
+If you'd like to know what else the team is currently considering or actively working on, refer to the https://github.com/orgs/OpenLiberty/projects/2[Open Liberty Roadmap] and the https://github.com/OpenLiberty/open-liberty/issues?q=is%3Aopen+is%3Aissue+label%3A%22In+Progress%22[in progress work items].
+
+The link:{url-about}[Open Liberty] 23.0.0.6-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<pkjwt, Private Key JWT authentication support for OpenID Connect clients>>
+* <<pkce, PKCE support for OpenID Connect clients>>
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/25333
+// Contact/Reviewer: ayoho
+// // // // // // // // 
+[#pkjwt]
+== Private Key JWT authentication support for OpenID Connect clients
+OpenID Connect clients in Liberty now support the `private_key_jwt` client authentication method with OpenID Connect token endpoints. 
+
+OpenID Connect clients are required to provide authentication data to the OpenID Connect provider when invoking the provider's token endpoint. Clients can authenticate using several different methods, but most of those methods require a client secret. The new `private_key_jwt` authentication method allows clients to make use of asymmetric keys to create signed JSON Web Tokens (JWTs) to authenticate instead of client secrets. OpenID Connect clients using this authentication method are no longer required to have a client secret.
```

Comment:
```suggestion
OpenID Connect clients are required to provide authentication data to the OpenID Connect provider when they invoke the provider's token endpoint. Clients can authenticate using several different methods, but most of those methods require a client secret. The new `private_key_jwt` authentication method enables clients to use asymmetric keys to create signed JSON Web Tokens (JWTs) to authenticate instead of client secrets. OpenID Connect clients using this authentication method are no longer required to have a client secret.
```